presto-main/src/main/java/com/facebook/presto/sql/gen/LambdaBytecodeGenerator.java

```diff
@@ -216,27 +213,7 @@ public static BytecodeNode generateLambda(
         }
 
         if (MethodHandle.class.equals(lambdaInterface)) {
-            // generate captured lambda expression as MethodHandle
-            List<BytecodeExpression> bytecodeCaptureVariables = bytecodeCaptureVariableBuilder.build().stream()
-                    .map(variable -> variable.cast(Object.class))
-                    .collect(toImmutableList());
-
-            Variable functionVariable = scope.createTempVariable(MethodHandle.class);
-            block.append(context.generate(lambda));
-            block.append(
-                    new IfStatement()
-                            .condition(wasNull)
-                            // ifTrue: do nothing i.e. Leave the null MethodHandle on the stack, and leave the wasNull variable set to true
-                            .ifFalse(
-                                    new BytecodeBlock()
-                                            .putVariable(functionVariable)
-                                            .append(invokeStatic(
-                                                    MethodHandles.class,
-                                                    "insertArguments",
-                                                    MethodHandle.class,
-                                                    functionVariable,
-                                                    constantInt(0),
-                                                    newArray(type(Object[].class), bytecodeCaptureVariables)))));
+            throw new UnsupportedOperationException();
```

Comment:
Please also search for other occurrences of `MethodHandle` in this pull request, and make adjustments in this last commit accordingly.